### ISSUE TITLE
Bulkloader will not load

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#2386]
Records marked to load return the error "b_build_keys_table: b_build_keys_table:"

I thought at first this might be related to my adding a column for ID confidence, but I've removed that and still get the error. 

Hoping to get these records loaded for loan processing by tomorrow.

**Comment by dustymc** [comment]
The accession number you provided isn't unique and therefore isn't resolvable.

**Comment by campmlc** [comment]
We should add this to the list of error interpretations. Absolutely not obvious.
Isn't there a way to prevent Arctos from accepting duplicate accession numbers?

**Comment by dustymc** [comment]
No, that's not obvious - but this is also the first time I'm aware of it happening.

Arctos allows duplicate accession (and loan and I believe borrow) numbers because someone's asked for that. That's easy enough to prevent from a technical perspective.
